docs/configuration/v1.x/metrics/azure-sql-database.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Azure SQL Database Declaration
+---
+
+## Azure SQL Database - ![Availability Badge](https://img.shields.io/badge/Available%20Starting-v1.0.0-green.svg)
```

Comment:
This would be shipped in v1.1.0
```suggestion
## Azure SQL Database - ![Availability Badge](https://img.shields.io/badge/Available%20Starting-v1.1.0-green.svg)
```

Reply:
np - I almost changed that actually, but I meant to ask you about it and just forgot. Sorted.